### PR TITLE
[SIG-3338] Add Map selection drawer

### PR DESCRIPTION
**Pull request by MauritsR**
### Additions

- Adds drawer showing list of selected containers
- Adds close button to selector map

### Changes

- Changes legend panel to render on above main panel

### CODE REVIEW

File: src/signals/incident/components/form/ContainerSelect/ContainerList/ContainerList.test.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { render, screen } from '@testing-library/react';
 import ContainerList from './ContainerList';
+import type { ContainerListProps } from './ContainerList';
 
-import { ContainerSelectProvider } from '../ContainerSelectContext';
 import { withAppContext } from 'test/utils';
 
 describe('signals/incident/components/form/ContainerSelect/ContainerList', () => {
-  const selection = [
+  const selection: ContainerListProps['selection'] = [
```
Comment (aditudorache) [inline src/signals/incident/components/form/ContainerSelect/ContainerList/ContainerList.test.tsx:9]: Why on `Item[]` ?

File: src/signals/incident/components/form/ContainerSelect/LegendToggleButton/LegendToggleButton.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const ICON_SIZE = 20;
+
+export interface LegendToggleButtonProps {
+  isRenderingLegendPanel: boolean;
+  onClick: () => void;
+}
+
+const StyledButton = styled(Button)`
+  box-sizing: border-box;
```
Comment (aditudorache) [inline src/signals/incident/components/form/ContainerSelect/LegendToggleButton/LegendToggleButton.tsx:17]: Why is this needed? Is not the default value?

File: src/signals/incident/components/form/ContainerSelect/Selector/Selector.test.tsx [hunk 3]
```diff
@@ -15,47 +15,74 @@ describe('signals/incident/components/form/ContainerSelect/Selector', () => {
...
   });
 
-  it('should call update when adding container', async () => {
+  it('should call update when adding container', () => {
     render(withContainerSelectContext(<Selector />));
     expect(contextValue.update).not.toHaveBeenCalled();
 
-    const element = await screen.findByText(/container toevoegen/i);
-    if (element) fireEvent.click(element);
+    const addContainerButton = screen.getByText(/containers toevoegen/i);
```
Comment (aditudorache) [inline src/signals/incident/components/form/ContainerSelect/Selector/Selector.test.tsx:35]: The async construction was added to avoid some nasty error in the console. Why is it removed? 
image

**Review by aditudorache — CHANGES_REQUESTED** [review]
Great Job! Works good. Just a couple of remarks.

File: src/signals/incident/components/form/ContainerSelect/ContainerList/ContainerList.test.tsx (hunk 1 above)
Comment (MauritsR) [reply src/signals/incident/components/form/ContainerSelect/ContainerList/ContainerList.test.tsx:9]: Changed to `Item[]`

File: src/signals/incident/components/form/ContainerSelect/LegendToggleButton/LegendToggleButton.tsx (hunk 2 above)
Comment (MauritsR) [reply src/signals/incident/components/form/ContainerSelect/LegendToggleButton/LegendToggleButton.tsx:17]: Discussed and resolved

File: src/signals/incident/components/form/ContainerSelect/Selector/Selector.test.tsx (hunk 3 above)
Comment (MauritsR) [reply src/signals/incident/components/form/ContainerSelect/Selector/Selector.test.tsx:35]: Fixed

**Review by aditudorache — APPROVED** [review]
Nice!
